packages/scrollbar/__tests__/scrollbar.spec.ts

```diff
@@ -110,4 +111,49 @@ describe('ScrollBar', () => {
 
     expect(wrapper.find('.el-scrollbar__wrap').attributes('style')).toContain('max-height: 200px;')
   })
+
+  test('should render always props', async () => {
+    const outerHeight = 200
+    const innerHeight = 500
+    const wrapper = _mount(`
+      <el-scrollbar height="${outerHeight}px" always>
+        <div style="height: ${innerHeight}px;"></div>
+      </el-scrollbar>
+    `)
+
+    expect(wrapper.find('.el-scrollbar__bar').attributes('style')).toBeFalsy()
```

Comment:
I think you need to test the existence of scrollbar instead of its style

`wrapper.find('.el-scrollbar__bar').exists()`